libs/parentParsers.py

```diff
@@ -130,18 +128,12 @@
 machinepoolParser.add_argument(
     '--machinepool-replicas',
     type=int,
-    default=2,
-    help='Number of hosts to create on the machinepool. Default: 2')
+    default=3,
+    help='Number of hosts to create on the machinepool. Default: 3')
 machinepoolParser.add_argument(
-    '--machinepool-wait',
-    dest='machinepool_wait',
-    action='store_true',
-    help='After creating the machinepool, wait until nodes to come up')
-machinepoolParser.add_argument(
-    '--machinepool-wait-cycles',
-    default=60,
+    '--machinepool-wait-time',
     type=int,
-    help='Number of 5 seconds wait cycles until halts the waiting')
+    help='Waiting time for hosts to come up')
```

Comment:
Waiting time in m?s? or should this be of a time type where they can specify 10m or 1h etc

Reply:
Seconds, updated